tests/api-mongo.test.js

```diff
@@ -412,3 +416,26 @@ describe('MongoDB Words', function () {
     });
   });
 });
+
+describe('API Requests For Home Directory "/"', () => {
+  describe('/undefinedRoute', () => {
```

Comment:
I don't think you need these `describes` you can just have `the top-level `describe` and then inside of that you can have the two `it` tests